docs/components/Layout.js

```diff
@@ -0,0 +1,75 @@
+// @flow strict
+import { useEffect, Fragment, type Node } from 'react';
+import { Box, Divider, ColorSchemeProvider, Link, Text } from 'gestalt';
+import { useRouter } from 'next/router';
+import Header from './Header.js';
+import Navigation from './Navigation.js';
+import { AppContextProvider, AppContextConsumer } from './appContext.js';
+import { NavigationContextProvider } from './navigationContext.js';
+
+type Props = {|
+  children?: Node,
+|};
+
+export default function App({ children }: Props): Node {
+  const router = useRouter();
+
+  useEffect(() => {
+    const handleRouteChange = () => {
+      if (!window.gtag) {
+        return;
+      }
+      window.gtag('config', 'UA-12967896-44', {
```

Comment:
what is this doing?

Reply:
google analytics tag? we could add a comment there.

Reply:
@AlbertCarreras added a comment.